Task70/YandexPages/LoginPage.cs

```diff
@@ -0,0 +1,31 @@
+﻿using OpenQA.Selenium;
+using SeleniumExtras.PageObjects;
+
+namespace Task70.YandexPages
+{
+    class LoginPage
```

Comment:
Please, add PageBase.cs class to this task. It will allow to store common things like driver variable, constructors etc. 
class LoginPage : PageBase